cupy/core/internal.pxd

```diff
@@ -29,3 +29,5 @@ cpdef vector.vector[Py_ssize_t] infer_unknown_dimension(
 cpdef slice complete_slice(slice slc, Py_ssize_t dim)
 
 cpdef tuple complete_slice_list(list slice_list, Py_ssize_t ndim)
+
+cpdef int clp2(size_t x)
```

Comment:
`int` should be `size_t`, otherwise it will not support arrays more than 2 GiB.

Reply:
Thanks!